projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/overview/entity/Contact.kt

```diff
@@ -98,6 +99,11 @@ class Contact(
     @Column(name = "soft_deleted", columnDefinition = "NUMBER", nullable = false)
     val softDeleted: Boolean = false
 ) {
+    var partitionAreaId: Long = 0
+
+    @CreatedBy
```

Comment:
This annotation won't do anything without the auditing entity listener annotation, probably worth removing it if you only need to read the createdByUserId - also can these two fields be `val`, or do we change them somewhere?

Reply:
updated